docs/getting_started/tutorial.md

```diff
@@ -141,10 +142,13 @@ Edit `home/models.py` as follows, to add a `body` field to the model:
 from django.db import models
 
 from wagtail.models import Page
+
+# add these new imports
 from wagtail.fields import RichTextField
 from wagtail.admin.panels import FieldPanel
 
 
+# replace the existing class definition of 'HomePage' with this:
```

Comment:
The intention here is that you should be able to copy-paste everything. Those comments wouldn’t be desirable to preserve in this code (they don’t meant anything outside the context of the step-by-step tutorial), so I don’t think this helps.

```suggestion
from wagtail.models import Page
from wagtail.fields import RichTextField
from wagtail.admin.panels import FieldPanel


```